tmt/steps/report/reportportal.py

```diff
@@ -1,152 +1,221 @@
+                                              "startTime": result.starttime})
+                self.handle_response(response)
+                item_uuid = yaml_to_dict(response.text).get("id")
+                assert item_uuid is not None
+                self.info("uuid", item_uuid, "yellow", shift=1)
+
+                # For each log
+                for i, log_path in enumerate(result.log):
+                    try:
+                        log = self.step.plan.execute.read(log_path)
+                    except tmt.utils.FileError:
+                        continue
+
+                    level = "INFO" if log_path == result.log[0] else "TRACE"
+                    status = self.TMT_TO_RP_RESULT_STATUS[result.result]
+
+                    # Upload log
+                    response = session.post(url=f"{url}/log/entry",
+                                            headers=headers,
+                                            json={"message": log,
+                                                  "itemUuid": item_uuid,
+                                                  "launchUuid": launch_uuid,
+                                                  "level": level,
+                                                  "time": result.endtime})
+                    self.handle_response(response)
+
+                    # Write out failures
+                    if i == 0 and status == "FAILED":
+                        message = result.failures(log)
+                        response = session.post(url=f"{url}/log/entry",
+                                                headers=headers,
+                                                json={"message": message,
+                                                      "itemUuid": item_uuid,
+                                                      "launchUuid": launch_uuid,
+                                                      "level": "ERROR",
+                                                      "time": result.endtime})
+                        self.handle_response(response)
+
+                # Finish the test item
+                response = session.put(url=f"{url}/item/{item_uuid}",
+                                       headers=headers,
+                                       json={
+                                           "launchUuid": launch_uuid,
+                                           "endTime": result.endtime,
+                                           "status": status})
+                self.handle_response(response)
+                launch_time = result.endtime
+
+            # Finish the launch
+            response = session.put(url=f"{url}/launch/{launch_uuid}/finish",
+                                   headers=headers,
+                                   json={"endTime": launch_time})
+            self.handle_response(response)
+            link = yaml_to_dict(response.text).get("link")
+            self.info("URL", link, "magenta")
```

Comment:
```suggestion
            self.info("url", link, "magenta")
```

Let's keep all keys consistent in lowercase.